packages/vapor/scss/tables/table-loading.scss

```diff
@@ -29,10 +29,6 @@ td.table-card-loading {
             width: 55%;
             min-width: 55%;
         }
-
-        & p:not(:first-child) .card-cell-content-loading {
-            @extend .bg-grey-1;
```

Comment:
Won't that break the current behavior? I think something like the following would be safer:

```scss
    & p:not(:first-child) .card-cell-content-loading {
        background-color: var(--grey-1);
    }
```